### DISCUSSION TITLE
Docs: Document how to build and use locally.

### DISCUSSION THREAD

**Discussion by jefftriplett** [djangopackages/djangopackages#987] [Docs]
High level notes:

```shell
docker-compose --profile=docs up
```

Open http://localhost:4000

**Comment by jefftriplett** [comment]
I'm leaning towards a docker compose `docs` profile might be the easier option. Then everything stays in docker.

**Comment by saadmk11** [comment]
I think we should consider having both options because if someone just wants to contribute to the documentation they might not want to use docker for that.
